components/pkg-export-kubernetes/src/manifest.rs

```diff
@@ -151,13 +154,20 @@ impl Manifest {
                       environment })
     }
 
-    /// Generates the manifest as a string and writes it to `write`.
+    /// Generates manifest as a string and writes it to `write`.
```

Comment:
Minor bit that happened when moving things around - please ignore this (I don't think it's worth another commit to correct it back to what it was, the meaning and intention is still clear)